hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/OzoneManagerServiceProviderImpl.java

```diff
@@ -539,7 +554,7 @@ boolean innerGetAndApplyDeltaUpdatesFromOM(long fromSequenceNumber,
    * full snapshot from Ozone Manager.
    */
   @VisibleForTesting
-  public boolean syncDataFromOM() {
+  public boolean syncDataFromOM() throws IOException {
```

Comment:
Mention in the JavaDoc that it throws an `IOException`.

Reply:
Yes , sure.. thanks for pointing out.